### PR TITLE
Tag Grafana dashboards

### LINKED ISSUE #208: Label and organize Grafana dashboards
We currently deploy all dashboards at the 'root'. Adding labels, and putting them in folders, would be good.

### PR DESCRIPTION
**Pull request by alxf**
Fix #208

See: #249 
See: https://github.com/scality/metalk8s/pull/249

### CODE REVIEW

File: roles/kube_prometheus/hack/fix-dashboard.py [hunk 1]
```diff
@@ -66,6 +66,14 @@ def main(dashboard_data, output, filename, title=None):
                 annotation['datasource'] = source_template
                 change = True
 
+    sys.stderr.write("{}\n".format(tags))
```
Comment (NicolasT) [inline roles/kube_prometheus/hack/fix-dashboard.py]: Huh? Debugging leftover?

File: roles/kube_prometheus/hack/fix-dashboard.py [hunk 2]
```diff
@@ -66,6 +66,14 @@ def main(dashboard_data, output, filename, title=None):
...
+    if tags is not None:
+        # reset initial tags list
+        dashboard['tags'] = []
```
Comment (NicolasT) [inline roles/kube_prometheus/hack/fix-dashboard.py]: I'd still write the one-line comprehension from #249 but sure, ok :wink:

**Review by NicolasT — CHANGES_REQUESTED** [review]
Think there's a debugging line left?

Also, needs a ChangeLog entry...

File: roles/kube_prometheus/hack/fix-dashboard.py (hunk 1 above)
Comment (alxf) [reply roles/kube_prometheus/hack/fix-dashboard.py]: good catch!

File: roles/kube_prometheus/hack/fix-dashboard.py (hunk 2 above)
Comment (alxf) [reply roles/kube_prometheus/hack/fix-dashboard.py]: mm i think i miss a commit, i made the one line as you suggested.

**Review by NicolasT — APPROVED** [review]
LGTM

**Review by NicolasT — CHANGES_REQUESTED** [review]
Ugh hold on

File: ChangeLog.rst [hunk 3]
```diff
@@ -9,6 +9,8 @@ Features added
 
 :ghpull:`222` - update Elasticsearch and `node_exporter` dashboards
 
+:ghpull:`208` - tag Grafana dashboards (:ghissue:`251`)
```
Comment (NicolasT) [inline ChangeLog.rst]: It's the other way around: issue is #208, PR is #251.

**Review by NicolasT — APPROVED** [review]

**Pull request merged**